core/templates/dev/head/pages/admin/admin.html

```diff
@@ -82,6 +82,7 @@
     {% include 'pages/footer_js_libs.html' %}
 
     <script src="{{TEMPLATE_DIR_PREFIX}}/app.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/stateful/FocusManagerService.js"></script>
```

Comment:
If moving this next to where all '/services' files are imported won't break anything, then let's move it (maybe to line #109, I think) ?